go.mod

```diff
@@ -112,12 +112,12 @@ require (
 	golang.org/x/crypto v0.14.0
 	golang.org/x/exp v0.0.0-20230817173708-d852ddb80c63
 	golang.org/x/net v0.17.0
-	golang.org/x/oauth2 v0.6.0
+	golang.org/x/oauth2 v0.7.0
 	golang.org/x/sync v0.3.0
 	golang.org/x/sys v0.13.0
 	golang.org/x/time v0.3.0
-	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1
-	google.golang.org/grpc v1.55.0
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230526203410-71b5a4ffd15e
```

Comment:
This swap for `genproto` was brought in by the pinning of `github.com/envoyproxy/go-control-plane` to `v0.11.1`. That pin was motivated by `google.golang.org/grpc` bringing it to `v0.11.1-0.20230524094728-9239064ad72f` during the initial bump; pinning within the same patch line is preferable to a SHA dep.